tests/util/dev-node.ts

```diff
@@ -80,6 +80,9 @@ export async function startMoonbeamDevNode(
     ETHAPI_CMD != "" ? `${ETHAPI_CMD}` : `--ethapi=txpool`,
     `--no-hardware-benchmarks`,
     `--no-telemetry`,
+    `--reserved-only`,
+    `--no-grandpa`,
+    `--database=paritydb`,
```

Comment:
Whilst these don't speed up execution, setup or teardown significantly; they should reduce overhead (by reducing footprint) and increase the reliability of running dev tests locally in parallel.

Reply:
not sure about the paritydb howerver (specially because it doesn't support some functionality required for Frontier SQL maintainance)

Reply:
Ok lets remove for time being then.